src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+                  {item.divider}
+                </div>
+              );
+            } else {
+              const hasHint = item.hint && item.hint.children ? true : false;
+              const itemHintId = `${id}-item-hint`;
+              let labelClasses = "govuk-radios__label";
+              labelClasses +=
+                item.label && item.label.className ? item.label.className : "";
+              const conditionalClasses =
+                "govuk-radios__conditional" + item.checked
+                  ? ""
+                  : " govuk-radios__conditional--hidden";
+
+              return (
+                <React.Fragment key={index}>
+                  <div className="govuk-radios__item">
+                    <input
+                      className="govuk-radios__input"
+                      id={id}
+                      name={name}
+                      type="radio"
+                      value={item.value}
+                      checked={item.checked ? true : undefined}
+                      disabled={item.disabled ? true : undefined}
+                      data-aria-controls={
+                        item.conditional && item.conditional.html
+                          ? `conditional-${id}`
+                          : undefined
+                      }
+                      aria-describedBy={hasHint ? itemHintId : undefined}
+                      {...radioExtraAttributes}
+                      onChange={
+                        onChange !== undefined
+                          ? (): void => {
+                              onChange(item.value as string);
+                            }
+                          : undefined
+                      }
+                    />
+                    <Label
+                      className={labelClasses}
+                      labelFor={id}
+                      {...item.label}
+                    >
+                      {item.label && item.label.children}
+                    </Label>
+                    {hasHint && (
+                      <Hint
+                        id={itemHintId}
+                        className="govuk-radios__hint"
+                        {...item.hint}
+                      >
+                        {item.hint && item.hint.children}
+                      </Hint>
+                    )}
+                  </div>
+                  {item.conditional && item.conditional.html && (
+                    <div
+                      className={conditionalClasses}
+                      id={`conditional-${id}`}
+                    >
+                      {item.conditional.html}
+                    </div>
+                  )}
+                </React.Fragment>
+              );
+            }
+          }
+        )}
+      </div>
+    </>
+  );
+
+  return (
+    <FormGroup {...formGroup}>
+      {fieldset ? (
+        <Fieldset
+          describedBy={describedBy.length > 0 ? describedBy : undefined}
```

Comment:
If you made `designedBy` an array instead, this could be `designedBy.join(" ")` (and avoid those ugly trailing spaces).

Reply:
You could also use `classnames` for this.

Reply:
I think we need this to be undefined if not set so I couldn't use classnames but have used an array to get around spacing issues like you suggested.